package.json

```diff
@@ -6,8 +6,10 @@
   { "type"      : "git"
   , "url"       : "git://github.com/tim-smart/node-closure.git"
   }
+, "dependencies": {
+    "google-closure-compiler": "x"
```

Comment:
Should probably [lock to a specific version](https://www.npmjs.com/package/google-closure-compiler) per update so the minifier (and possible bugs changes) don't change under this package's feet.